### DISCUSSION TITLE
Welcome to style-station-plugin Discussions!

### DISCUSSION THREAD

**Discussion by MaxBazarov** [MaxBazarov/style-station-plugin#1] [General]
Hello. 
Ask you questions here.

**Comment by creativebashar** [comment]
Hello @MaxBazarov 

Thank you for a great plugin :)
I'm not a technical guy, but have a few skills to get me through.

A few enhancements would be great:

1. Support gradients background colors, as it exports them as but doesn't import them back.
2. Support effects

Thank you again.

**Comment by MaxBazarov** [comment]
Hello!

Thanks for the feedback.
I’m researching gradients right now.

Will keep you informed.
